### DISCUSSION TITLE
Can't parse config: "input in flex scanner failed"

### DISCUSSION THREAD

**Discussion by liamriley1** [strongswan/strongswan#1195] [Q&A]
When I run `systemtl start strongswan` it fails, and the logs show:
```
Started strongSwan IPsec IKEv1/IKEv2 daemon using ipsec.conf.
input in flex scanner failed
```
My ipsec.conf file:
```
config setup
    strictcrlpolicy=yes
    uniqueids = no
    charondebug="ike 4, knl 4, cfg 3, chd 4"

conn pfsense
    right=@ikev2.vpn.mydomain.com
    # This should match the `leftid` value on your server's configuration
    rightid=ikev2.vpn.mydomain.com
    rightsubnet=0.0.0.0/0
    rightauth=pubkey
    leftsourceip=%config
    leftid=user@mydomain.com
    leftauth=eap-mschapv2
    eap_identity=%identity
    auto=start
```

I have tried checking the file for any weird newline characters or syntax errors, but I can't really see anything wrong, or any errors stating what exactly is at fault with the config file. Thanks in advance for any help.

**Comment by strongX509** [comment]
The spaces in front and after the **=** sign?
````
    uniqueids = no
````
I haven't been using `ipsec.conf` and its legacy syntax in the last ten years so I really don't know.

**Reply by liamriley1** [reply]
Thanks for your help. Tried that change but still getting the same error. When you say that you haven't used it in ten years, is there a alternative way to configure strongswan instead?

**Reply by strongX509** [reply]
Well, the standard way is using the `swanctl.conf` configuration file together with the `swanctl` command line tool. See

https://docs.strongswan.org/docs/5.9/swanctl/swanctlConf.html

and

https://docs.strongswan.org/docs/5.9/config/quickstart.html
